monolog/composer.json

```diff
@@ -10,8 +10,13 @@
 	  "role": "Developer"
 	}
   ],
+  "autoload": {
+    "psr-4": {
+	  "Friendica\\Addon\\Monolog\\": "src/"
```

Comment:
Indentation mismatch, your IDE is misbehaving I suppose!

Reply:
Yeah, it creates TAB all time like for PHP files. I have to investigate it, but for this PR, I just replace it with spaces;-)